GitUI/CommandsDialogs/FormVerify.LostObject.cs

```diff
@@ -22,15 +22,40 @@ private enum LostObjectType
 
         private sealed class LostObject
         {
+            private static GitArgumentBuilder _LogBuilder;
+            private static GitArgumentBuilder _TagBuilder;
```

Comment:
These should be `_logBuilder` and `_tagBuilder`

Reply:
Sure.